README.md

```diff
@@ -63,6 +79,9 @@ Note: If you have cloned the repo previously make sure you remove your node_modu
 
 ### Building for nodejs:
 Be sure you have python2.7 as the default python. 3.x won't work, and it's not enough to use `--python=python2.7` as parameter to npm.
+```
+brew install python@2
```

Comment:
```suggestion
brew install python@2     # or use any other means of installing 
```

Reply:
This is fixed here 
https://github.com/realm/realm-js/pull/2473/commits/857afe7270d9070b6868850170d33785b788edb8

So I will resolve this one